src/ring_theory/adjoin/power_basis.lean

```diff
@@ -54,7 +54,7 @@ end
 
 /-- The power basis `1, x, ..., x ^ (d - 1)` for `K[x]`,
 where `d` is the degree of the minimal polynomial of `x`. -/
-noncomputable def adjoin.power_basis {x : S} (hx : _root_.is_integral K x) :
+@[simps gen dim] noncomputable def adjoin.power_basis {x : S} (hx : _root_.is_integral K x) :
```

Comment:
Is there a reason you don't also let it generate `basis`?

Reply:
it doesn't create a good lemma:

```lean
(adjoin.power_basis hx).basis = adjoin.power_basis_aux hx
```

Reply:
What if you stick `simps` on `power_basis_aux` too?

Reply:
same; I feel like it doesn't particularly matter anyways as we have the `basis_eq_pow` field

Reply:
In that case, we should probably globally turn off `basis` with simps, using something like
```lean
-- not needed, since `basis_eq_pow` covers it
initialize_simps_projections power_basis (-basis)
```

Reply:
alright, let's see if something blows up. cc @Vierkantor, do you think this is a good idea?

Reply:
Makes sense to me!

Reply:
Okay, so I reverted these changes. I still think we don't want this result for the current basis (that is, I think it could be `irreducible` without harm) but for some more technical power-bases like `map` it can be useful. It'd be nice if we could make the "default" mode not write `basis`, but we can request it by hand; is that something you can do with `simps`?

Reply:
According to the [`simps` documentation](https://leanprover-community.github.io/mathlib_docs/tactic/simps.html#initialize_simps_projections_cmd):

> You can disable a projection by default by running initialize_simps_projections equiv (-inv_fun) This will ensure that no simp lemmas are generated for this projection, *unless this projection is explicitly specified by the user*.

So I think `initialize_simps_projections power_basis (-basis)` plus `@[simps basis] def map` would be the approach here.

Reply:
@ericrbg do you agree?

Reply:
Yes, was just asleep - I'll sort this :)